structure_tests/tests/debian_failure_test.yaml

```diff
@@ -0,0 +1,34 @@
+schemaVersion: '1.0.0'
```

Comment:
Since this is yaml now, could you add some comments in here explaining what this is for?

Reply:
It's all in the README, but we really should be testing the newest schema version, so I'm going to fix that.